{"schemaVersion":{"major":0,"minor":3,"patch":0},"metadata":{"modules":[{"name":"BeyondIdentityEmbedded"}],"role":"symbol","externalID":"s:22BeyondIdentityEmbedded17RedeemOtpResponseO06failedE0yAcA0e9ChallengeF0VcACmF","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"failedOtp","kind":"identifier"},{"kind":"text","text":"("},{"preciseIdentifier":"s:22BeyondIdentityEmbedded20OtpChallengeResponseV","text":"OtpChallengeResponse","kind":"typeIdentifier"},{"kind":"text","text":")"}],"title":"RedeemOtpResponse.failedOtp(_:)","symbolKind":"case","roleHeading":"Case"},"hierarchy":{"paths":[["doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded","doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded\/RedeemOtpResponse"]]},"variants":[{"paths":["\/documentation\/beyondidentityembedded\/redeemotpresponse\/failedotp(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"failedOtp","kind":"identifier"},{"kind":"text","text":"("},{"kind":"typeIdentifier","identifier":"doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded\/OtpChallengeResponse","preciseIdentifier":"s:22BeyondIdentityEmbedded20OtpChallengeResponseV","text":"OtpChallengeResponse"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["iOS"]}],"kind":"declarations"}],"abstract":[{"text":"A response returned on failure to authenticate with the provided OTP code.","type":"text"},{"text":" ","type":"text"},{"text":"Use ","type":"text"},{"type":"codeVoice","code":"OtpChallengeResponse.url"},{"type":"text","text":" to initiate a retry on either "},{"type":"codeVoice","code":"redeemOtp"},{"type":"text","text":" or "},{"type":"codeVoice","code":"authenticateOtp"}],"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded\/RedeemOtpResponse\/failedOtp(_:)"},"references":{"doc://BeyondIdentityEmbedded/documentation/BeyondIdentityEmbedded/OtpChallengeResponse":{"url":"\/documentation\/beyondidentityembedded\/otpchallengeresponse","title":"OtpChallengeResponse","identifier":"doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded\/OtpChallengeResponse","abstract":[{"text":"A response returned if the SDK requires an OTP.","type":"text"}],"kind":"symbol","type":"topic","navigatorTitle":[{"text":"OtpChallengeResponse","kind":"identifier"}],"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"OtpChallengeResponse"}],"role":"symbol"},"doc://BeyondIdentityEmbedded/documentation/BeyondIdentityEmbedded":{"type":"topic","abstract":[{"type":"text","text":"Passwordless Authentication for Swift"}],"url":"\/documentation\/beyondidentityembedded","title":"BeyondIdentityEmbedded","role":"collection","identifier":"doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded","kind":"symbol"},"doc://BeyondIdentityEmbedded/documentation/BeyondIdentityEmbedded/RedeemOtpResponse":{"type":"topic","navigatorTitle":[{"kind":"identifier","text":"RedeemOtpResponse"}],"abstract":[{"text":"A response returned after attempting to redeem an OTP","type":"text"}],"url":"\/documentation\/beyondidentityembedded\/redeemotpresponse","title":"RedeemOtpResponse","role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"RedeemOtpResponse"}],"identifier":"doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded\/RedeemOtpResponse","kind":"symbol"},"doc://BeyondIdentityEmbedded/documentation/BeyondIdentityEmbedded/RedeemOtpResponse/failedOtp(_:)":{"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"failedOtp"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"OtpChallengeResponse","preciseIdentifier":"s:22BeyondIdentityEmbedded20OtpChallengeResponseV"},{"text":")","kind":"text"}],"title":"RedeemOtpResponse.failedOtp(_:)","type":"topic","abstract":[{"text":"A response returned on failure to authenticate with the provided OTP code.","type":"text"},{"type":"text","text":" "},{"type":"text","text":"Use "},{"type":"codeVoice","code":"OtpChallengeResponse.url"},{"type":"text","text":" to initiate a retry on either "},{"type":"codeVoice","code":"redeemOtp"},{"text":" or ","type":"text"},{"code":"authenticateOtp","type":"codeVoice"}],"kind":"symbol","identifier":"doc:\/\/BeyondIdentityEmbedded\/documentation\/BeyondIdentityEmbedded\/RedeemOtpResponse\/failedOtp(_:)","url":"\/documentation\/beyondidentityembedded\/redeemotpresponse\/failedotp(_:)"}}}